opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -153,6 +153,12 @@ namespace Opm {
         const std::map<int, VFPInjTable>& getVFPInjTables() const;
         size_t getNumPhases() const;
 
+        // The saturation function family.
+        // If SWOF and SGOF are specified in the deck it return 1
+        // If SWFN, SGFN and SOF3 are specified in the deck it return 2
+        // If keywords are missing or mixed, an error is given.
+        const size_t getSaturationFunctionFamily() const;
```

Comment:
> [...] you can argue that this should be done into opm-parser [ ...] must be a canonical representation for all the families. 

That has been my "wet dream" for the saturation tables in the parser - but I have no clue how large a job that would be.

I have no strong feelings if the switch should be an enum or a bool - but `size_t` is in my opinion even worse than int.